modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/ElementSetList.es.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {EmptyState, FieldType, SearchUtils} from 'data-engine-taglib';
+import {
+	elementSetAdded,
+	useForm,
+	useFormState,
+} from 'dynamic-data-mapping-form-renderer';
+import React from 'react';
+
+const EmptyPanel = ({searchTerm}) => (
+	<div className="mt-2">
+		<EmptyState
+			emptyState={{
+				description: Liferay.Language.get(
+					'there-are-no-element-sets-yet'
+				),
+				title: Liferay.Language.get('there-are-no-element-sets'),
+			}}
+			keywords={searchTerm}
+			small
+		/>
+	</div>
+);
+
+const ElementSetList = ({elementSets, searchTerm = '', ...context}) => {
```

Comment:
We can move this file to the respective plugin folder since it will only be used there. It also creates the sense that this component belongs only to the plugin and is lazy loaded only in the Sidebar.

`panels/field-sidebar/components`